packages/flutter/test/widgets/focus_manager_test.dart

```diff
@@ -354,9 +354,13 @@ void main() {
       logs.clear();
     }, variant: KeySimulatorTransitModeVariant.all());
 
-    testWidgets('FocusManager ignores app lifecycle changes on Android.', (WidgetTester tester) async {
-      final bool shouldRespond = kIsWeb || defaultTargetPlatform != TargetPlatform.android;
-      if (shouldRespond) {
+    testWidgets('FocusManager ignores app lifecycle changes on Android and iOS.', (WidgetTester tester) async {
+      final bool mobileDevice = switch (defaultTargetPlatform) {
+        TargetPlatform.android || TargetPlatform.iOS => false,
+        TargetPlatform.fuchsia || TargetPlatform.linux => true,
+        TargetPlatform.windows || TargetPlatform.macOS => true,
+      };
+      if (kIsWeb || mobileDevice) {
```

Comment:
What do you think about removing this check, and instead specifying `Android` and `iOS` as the test variants, and skipping the web test? I think as the test stands now it is only running on `Android`.

Reply:
Can't believe I didn't notice that you can specify a test variant! Thanks for the suggestion.